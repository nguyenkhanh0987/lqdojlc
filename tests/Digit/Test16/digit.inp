100
591071658 759989865
240046463 868426169
559645557 616421068
853769974 991351332
791925933 984567663
413565015 921029589
804521603 897176191
973813191 989224707
124326636 635824238
302331135 487941347
27314130 249237718
148753475 272790485
931434847 945891315
609622965 716474282
306802999 673464949
12082917 618089099
956978288 977930359
827467549 830738095
615310874 971330711
364276664 676235895
395826099 669617097
95868214 794036636
427970496 834678271
915663267 979563020
557576027 876455056
820823020 832678127
166997980 460278778
75417410 86097250
279090851 895697577
992805701 992934880
397144404 595828450
879514566 947631237
717657751 729041474
12343498 467812465
849635263 857576294
319047394 361971833
331478632 621830894
874692834 883981795
826221381 849804619
841476541 968924105
33774712 232932168
261215282 887922167
61755232 498086910
680465364 996859195
10862474 936947263
138497536 994952374
244723603 392442283
761813486 907836349
735146721 875451111
812778459 963117871
709724426 950340288
274510622 393574867
697785535 790303785
612658862 813472646
346716439 382786161
12905663 321886280
419206053 896383152
638192196 831218944
201842591 344246302
773769874 913728817
515504809 689286246
417955661 814713188
7255283 669617698
453102846 478348487
172032944 632384250
693777918 828282660
482638967 888811640
737477210 826348600
935612829 940787652
644866213 930078901
749964543 826328344
444648044 683376307
434886820 937316726
547268678 729135581
357065365 655615809
36863182 408898080
122684048 242736559
814017007 914716914
374910200 459355620
894248065 935733573
890911141 944487598
704668846 911375541
393233554 412079411
242690644 683453975
734926655 881839508
883969574 919549752
812571371 842880739
498882942 766592007
519393075 813437608
556061100 902101052
465457219 527554763
52172847 95347565
602497449 938657056
156541366 298596159
542465836 592729218
275626200 326042602
301096238 489723683
488564139 636674254
679671745 958891542
383878516 530318448
