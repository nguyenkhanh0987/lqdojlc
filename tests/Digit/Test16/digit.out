15264603
76023137
7716631
9900226
12804118
51054946
6876938
1377298
76348677
26295975
42375363
23050762
666825
10452284
46333702
97637644
1603427
89107
28411736
39411634
31569264
96639806
43433550
3347715
30336875
540383
46959381
2865433
71815116
35297
25378360
4157194
489667
78629059
668348
5879298
39042377
1081990
1106514
8561414
38846961
74468993
75462613
25074363
124215414
104004628
25489517
12424299
11398999
9377744
17171920
20106707
7339416
18667648
6556864
57669179
50000978
17736564
21763479
11167187
19237900
43515752
102745171
4040850
67135005
10430436
40048988
7822744
134178
24001175
7040173
28952874
49688902
19599318
38094926
67847535
21577477
7161737
11286577
1560408
2586626
15927324
2326297
60571116
12052842
2304150
1296245
26425022
30077166
33376524
8317915
11088154
26977854
27543583
7486651
8219656
26635918
16344077
21492047
19004000
